theano/gpuarray/opt.py

```diff
@@ -590,8 +591,8 @@ def local_gpuaalloc(op, context_name, inputs, outputs):
 def local_gpuaallocempty(op, context_name, inputs, outputs):
     # We use _props_dict() to make sure that the GPU op know all the
     # CPU op props.
-    return GpuAllocEmpty(context_name=context_name,
-                         **op._props_dict())(*inputs)
+    dtype = op._props_dict().get('dtype')
+    return gpu_alloc_empty(dtype,context_name)(*inputs)
```

Comment:
The props dict is to help prevent error. You can do it like this to keep that.

props = op._props_dict()
props['context_name'] = context_name
return gpu_alloc_empty(*props)(*inputs)

so if w new props appear, an error will be raised.